lib/dshelper/txndswrap/txndswrap.go

```diff
@@ -1,6 +1,7 @@
 package txndswrap
```

Comment:
I had to fix our in-memory `go-datastore` impl to satisfy the new APIs now receiving ctx.